server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -183,7 +183,11 @@ private class ProGuardContext {
             envContext.put("env.LD_LIBRARY_PATH", "."); // Easier when running a standalone local without such a variable
 
             processExecutor.putEnv("DYNAMO_HOME", sdk.getAbsolutePath());
-            processExecutor.putEnv("JAVA_HOME", System.getenv("JAVA_HOME"));
+            String java_home = System.getenv("JAVA_HOME");
+            if (java_home != null)
+            {
+                processExecutor.putEnv("JAVA_HOME", java_home);
+            }
```

Comment:
We currently haven't setup the JAVA_HOME on the darwin server.
We'll need it for the Rive extension, since we are building jni stuff, and we need the java sdk (jni.h):

```
defold-rive/ext.manifest
    includes:   ["{{env.JAVA_HOME}}/include/linux", "{{env.JAVA_HOME}}/include"]
```